PWGGA/GammaConv/AliPrimaryPionCuts.h

```diff
@@ -196,6 +198,7 @@ class AliPrimaryPionCuts : public AliAnalysisCuts {
 	TH2F *fHistTPCdEdxafter; // TPC dEdx after cuts
 	TH2F *fHistTPCdEdxSignalbefore; //TPC dEdx signal before
 	TH2F *fHistTPCdEdxSignalafter; //TPC dEdx signal  after
+	TH2F *fHistoTOFSigbefore; // TOF signal after cuts
```

Comment:
In general histograms dont need to be streamed and you could have them commented as //! I think